src/components/molecules/__tests__/BalancesCalendar.test.tsx

```diff
@@ -0,0 +1,54 @@
+import { cleanup, getByText, render } from '@testing-library/react';
+import 'jest-dom/extend-expect';
+import moment from 'moment';
+import React from 'react';
+import GlobalWrapper from '../../../helpers/GlobalWrapper';
+import BalancesCalendar from '../BalancesCalendar';
+
+test('BalancesCalendar renders entries correct', () => {
+  const entries = [
+    {
+      date: '2019-06-01',
+      expense: 200000,
+      liquidity: 10000000,
+    },
+    {
+      date: '2019-06-08',
+      income: 200000,
+      liquidity: 10200000,
+    },
+    {
+      date: '2019-06-15',
+      income: 400000,
+      liquidity: 10600000,
+    },
+  ];
+
+  const month = moment(entries[0].date);
+
+  const { container, getByText } = render((
+    <GlobalWrapper>
+      <BalancesCalendar entries={entries} month={month} />
+    </GlobalWrapper>
+  ));
+
+  // Check that headers are rendered
+  const days = ['Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat', 'Sun'];
```

Comment:
See earlier comment. The result is the same, but this should respect locale, which I suppose is nice. And it's a bit DRYer.

https://momentjs.com/docs/#/i18n/listing-months-weekdays/

Reply:
See earlier comment on similar issue.